docs/getting-started/config/authentication.md

```diff
@@ -8,43 +8,48 @@ Audience: Admins or Developers
 
 ### Setting up authentication
```

Comment:
Can we give a little context on what we're doing before jumping in? It's been a minute since I did the doc rewrites, but I think I was trying to do a `Summary` section for each doc that outlines what you'll learn after reading it. 

Something like,
```
### Summary
We'll be walking you through how to setup authentication for your Backstage app using Github. After finishing this guide, you'll have both working authentication and users in your Backstage app to match to the users logging in!
```

Reply:
Done 👍